test/tests/functional/pbs_cray_suspend_resume.py

```diff
@@ -67,6 +67,8 @@ def test_exclusive_job_not_suspended(self):
         """
         If a running job is a job with exclusive placement then this job can
         not be suspended.
+        This test is checking for a loge message which is an unstable
```

Comment:
Minor comment: misspelled "log"

Reply:
@vccardenas & @lisa-altair I've removed the comment and made the log interface stable.

Reply:
@arungrover , thanks for updating the EDD and the test (removed comment and expecting exec_host to be set).  Looks good to me now.

Reply:
Ahh... It just never ends... :)
rectified in latest push